src/taskpane/outlook.ts

```diff
@@ -15,6 +15,6 @@ const insertText = async (text: string) => {
   } catch (error) {
     console.log("Error: " + error);
   }
-};
+}
 
 export default insertText;
```

Comment:
should remove export default insertText here. There's no need to use default export for code which isn't a react component.

Reply:
Done.